docs/changelog.rst

```diff
@@ -5,6 +5,7 @@ Changelog
 
 Changes in 0.9.X - DEV
 ======================
+- Request Support for $min, $max Field upate operators #863
```

Comment:
in my original issue report I incorrectly spelled `upate`... you may wish to update your change log entry accordingly.

` - Added support fo $min, $max Field update operators #863`